docs/extra_vars.rst

```diff
@@ -5,9 +5,9 @@ Extra vars for playbooks or job templates
 Extra vars needed by each playbook or job template should be explicitly
 specified in the extra_vars section under actions in a rulebook. The values of
 extra vars can be literals or references to rulebook variables. In addition to
-the user specified extra vars, the rulebook engine automatically inserts 
-fact/event (single match) or facts/events (multiple matches) under the top
-level key ansible_eda.
+the user specified extra vars, the rulebook engine automatically inserts
```

Comment:
Just a note that I am refactoring the docs about variables in #359 and this includes moving the extra_vars info to a new page, so I have incorporated this new text into that refactor. The extra_vars page will no longer exist after the refactor.

Reply:
ok. Thanks